packages/theme-generator/src/install-nrg-css.js

```diff
@@ -0,0 +1,60 @@
+import { execSync } from 'child_process';
+import { existsSync, readFileSync } from 'fs';
+import { resolve } from 'path';
+import { coerce, gte } from 'semver';
+
+export function getPackageFile() {
+  return JSON.parse(readFileSync(resolve('package.json'), 'utf-8'));
+}
+
+export function getDependenciesFromPackage() {
+  const pkg = getPackageFile();
+
+  const dependencies = {
+    ...pkg.dependencies,
+    ...pkg.devDependencies,
+  };
+
+  return dependencies;
+}
+
+export default async function () {
```

Comment:
If this script is called `install-nrg-css`, should it attempt to install the `@nrg-ui/css` package with the minimum version if it's not found, instead of just erroring?

Reply:
Right now that would be fine.
In the future, doing so will install far too low of a version.

I prefer to have the user think about which version they should install, even if it's less convenient.